### DISCUSSION TITLE
Document update cadence with name-suggestion-index.

### DISCUSSION THREAD

**Discussion by EvanCarroll** [openstreetmap/iD#9357] [Q&A]
I just landed a patch in name-suggestion-index. I wanted to see if it was already on iD. It was not. I don't see it documented in name-suggestion-index how often they release, but it seems once a week (as inferred here https://github.com/osmlab/name-suggestion-index/tags).

The package.json for iD shows name-suggestion-index is 

    "name-suggestion-index": "~6.0",

Which means any new release will get picked up. However the release schedule for [iD is much less frequent](https://github.com/openstreetmap/iD/releases). How long does it typically take for a patch to name-suggestion-index to land in iD. Because technically any new release of name-suggestion-index can get included if it's major version number is compat, I would expect it to be updated in the next week? Do the servers running production `npm install` (getting the newest version of name-suggestion-index periodically between releases)?

How does it take and what does it look like after you get a patch into NSI before it shows in iD?

**Comment by bhousel** [comment]
NSI releases happen about once a week, @arch0345 is doing a great job keeping it fresh!
https://github.com/osmlab/name-suggestion-index/tags

iD and RapiD are setup to fetch the latest 6.x data files from the jsDelivr CDN:
https://github.com/facebookincubator/RapiD/blob/32ea5d1639e0bfcc0d6c9cabaaff598d872294ed/modules/services/nsi.js#L42-L50

Our release steps include the commands to purge the CDN, so as soon as the release is published and those commands run, users will see the update the next time they start iD or RapiD.
